src/components/CreateProvider.svelte

```diff
@@ -0,0 +1,90 @@
+<script lang="ts">
+    import {dotApi, storeCurrentAction, transactionSigningAddress} from "$lib/stores";
+    import type {DotApi} from '$lib/storeTypes';
+    import {defaultDotApi} from "$lib/storeTypes";
+    import type {ApiPromise} from "@polkadot/api";
+    import {isLocalhost} from "$lib/utils";
+    import {submitCreateProvider} from "$lib/connections";
+    import TransactionStatus from "$components/TransactionStatus.svelte";
+    import {isFunction} from "@polkadot/util";
+    import {onMount} from "svelte";
+
+    let newProviderName = '';
+    let localDotApi: DotApi = defaultDotApi;
+    let web3FromSource;
+    let web3Enable;
+    let showTransactionStatus = false;
+    export let txnStatuses: Array<string> = [];
+    export let validAccounts = {}
+    export let signingAddress = '';
+    export let cancelAction;
+
+    onMount(async () => {
+        const extension = await import('@polkadot/extension-dapp');
+        web3FromSource = extension.web3FromSource;
+        web3Enable = extension.web3Enable;
```

Comment:
Likely a onMount need, but curious why we have to enable web3 everywhere instead of just the one time in `Connect.svelte`?

Reply:
I can try to pull it out, but this change should go in a follow-up since that's how we're doing it everywhere.